prog/weaveutil/datapath.go

```diff
@@ -50,3 +85,56 @@ func addDatapathInterface(args []string) error {
 	}
 	return odp.AddDatapathInterface(args[0], args[1])
 }
+
+// Helpers
+
+func checkMTU(dpname string, mtu int) bool {
+	var (
+		vpid   libodp.VportID
+		vpname string
+	)
+
+	// Create a dummy vxlan vport
+	for i := 0; i < 5; i++ {
```

Comment:
At least leave a comment.  The explanation for that one seems to be at https://github.com/weaveworks/weave/blob/master/router/fastdp.go#L95